frontend/src/components/Appbar/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface AppbarProps {
+  agreementName: string;
+  closeFn: () => void;
+}
```

Comment:
Let's move this into `Appbar.tsx` and remove Appbar dir (same thing we did with other components)